src/main/java/Duke.java

```diff
@@ -1,10 +1,201 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+
+
 public class Duke {
+
+    enum Type {
+        TODO, DEADLINE, EVENT
+    }
+
+    private static ArrayList<Task> taskList = new ArrayList<>();
+    private static boolean startDuke = true;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        Scanner sc = new Scanner(System.in);
+        
+        loadFile();
+
+        while (startDuke) {
+           initDuke(sc);
+        }
+    }
+
+    public static void loadFile() {
+        try {
+            taskList = DukeFile.loadFile();
+            for (Task t : taskList) {
+                System.out.println(t);
+            }
+        } catch (FileNotFoundException e) {
+            System.out.println(e);
+        }
+    }
+
+    public static void saveFile(ArrayList<Task> taskList) {
+        DukeFile.saveData(taskList);
+    }
+
+    public static void initDuke(Scanner sc) {
+        String[] userInput = sc.nextLine().split(" ", 2);
+    
+        try {
+            switch (userInput[0]) {
+                case "bye":
+                    System.out.println("Bye. Hope to see you again soon!");
+                    sc.close();
+                    startDuke = false;
+                    break;
```

Comment:
Switch-case should not be indented. Could edit coding style in Intellij to reduce having to manually check.